clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -65,4 +73,94 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:complexType name="server-side-config-spec">
+    <xs:sequence>
+      <xs:element name="default-resource" type="tc:resource-link-spec" minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation>
+            Specifies the default server-side storage resource to use for storing cache data.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="shared-pool" type="tc:shared-pool-spec" minOccurs="0" maxOccurs="unbounded">
+        <xs:annotation>
+          <xs:documentation>
+            Defines a pool of server-side storage resource to be shared amongst multiple caches.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+
+  <xs:complexType name="resource-link-spec">
+    <xs:attribute name="from" type="xs:string" use="required">
+      <xs:annotation>
+        <xs:documentation>
+          Reference to a server-side storage resource.
+        </xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+  </xs:complexType>
+
+  <xs:complexType name="shared-pool-spec">
+    <xs:complexContent>
+      <xs:extension base="eh:memory-type">
+        <xs:attribute name="name" type="xs:string" use="required">
+          <xs:annotation>
+            <xs:documentation>
+              Shared pool name.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="from" type="xs:string" use="optional">
```

Comment:
Did you consider using a global attribute (`xsd:attribute`) for `from`?  Seems like this `from` is the same "type" of reference as `from` abovee.